src/config/entities/configuration.ts

```diff
@@ -131,26 +132,40 @@ export default () => ({
       apiKey: process.env.INFURA_API_KEY,
     },
   },
-  typeorm: { autoLoadEntities: true, manualInitialization: true },
   db: {
-    postgres: {
-      host: process.env.POSTGRES_HOST || 'localhost',
-      port: process.env.POSTGRES_PORT || '5432',
-      database: process.env.POSTGRES_DB || 'safe-client-gateway',
-      schema: process.env.POSTGRES_SCHEMA || 'main', //@TODO: use this schema
-      username: process.env.POSTGRES_USER || 'postgres',
-      password: process.env.POSTGRES_PASSWORD || 'postgres',
-      ssl: {
-        enabled: process.env.POSTGRES_SSL_ENABLED?.toLowerCase() === 'true',
-        requestCert:
-          process.env.POSTGRES_SSL_REQUEST_CERT?.toLowerCase() !== 'false',
-        // If the value is not explicitly set to false, default should be true
-        // If not false the server will reject any connection which is not authorized with the list of supplied CAs
-        // https://nodejs.org/docs/latest-v20.x/api/tls.html#tlscreateserveroptions-secureconnectionlistener
-        rejectUnauthorized:
-          process.env.POSTGRES_SSL_REJECT_UNAUTHORIZED?.toLowerCase() !==
-          'false',
-        caPath: process.env.POSTGRES_SSL_CA_PATH,
+    migrator: {
+      executeMigrations:
+        process.env.DB_MIGRATIONS_EXECUTE?.toLowerCase() !== 'false',
+      numberOfRetries: process.env.DB_MIGRATIONS_NUMBER_OF_RETRIES ?? 5,
+      retryAfter: process.env.DB_MIGRATIONS_RETRY_AFTER ?? 1000, // Milliseconds
```

Comment:
Nit: besides the comment, could we rename this to `retryAfterMs/DB_MIGRATIONS_RETRY_AFTER_MS`, I think it's good to have the scale in the variable name as we do for other configuration properties. (A person changing this in the k8s/container configuration might not have access to this source code to read the comment and know the scale for some reason)

Reply:
Done
https://github.com/safe-global/safe-client-gateway/pull/1988/commits/378fdd1d0c485f4b3dbdba8a8ed0efc33d8a2121